programs/lfx-mentorship/2025/03-Sep-Nov/project_ideas.md

```diff
@@ -17,3 +17,42 @@
 ---
 
 ## Proposed Project ideas
+
+### KubeStellar
+
+#### KubeStellar UI Playwright Testing Implementation
+
+CNCF - KubeStellar: Implement End-to-End Testing Framework Using Playwright for KubeStellar UI (2025 Term 3)
+
+- **Description:**  
+  This project focuses on building a robust end-to-end testing framework for the KubeStellar UI using Playwright. The goal is to ensure seamless user experiences by automating critical workflows, enhancing test coverage, and catching UI issues early across various browsers and environments. This includes setting up Playwright from scratch, writing reusable and scalable test suites using the Page Object Model (POM), and integrating these tests into the CI/CD pipeline. Special focus will be given to WebSocket-based testing, authentication, and resource-heavy scenarios in a multi-cluster Kubernetes context.
+
+- **Expected Outcome:**
```

Comment:
Hi, thanks for this @btwshivam. Please don't add formatting to the template as I have to strip it all out again when I load it into the LFX platform.

@clubanderson, @MAVRICK-1, could you both please approve this PR or give a 👍 to confirm your participation?